PyRoute/Star.py

```diff
@@ -150,7 +153,7 @@ def __init__(self):
         self.raw_be = None
         self.im_be = None
         self.col_be = None
-        self.star_list = None
+        self.star_list_object = None
```

Comment:
I think you can keep this as just "star_list".

Reply:
I converted star_list to a property (to minimise blast damage outside Star class), so star_list_object holds the actual StarList object